autoextract_poet/adapters.py

```diff
@@ -0,0 +1,65 @@
+from types import MappingProxyType
+from typing import Any, KeysView, Iterator
+
+from itemadapter.adapter import AttrsAdapter
+
+from autoextract_poet.items import Item
+
+
+class AutoExtractAdapter(AttrsAdapter):
+    """
+    ``ItemAdapter`` for AutoExtract poet items. The advantage of
```

Comment:
Is it an advantage, or a different behavior?

Reply:
Modified here: https://github.com/scrapinghub/autoextract-poet/pull/13/commits/cd9e3d52fc3e80cd83aeae1a4a7fe81c4ef24a03 and here: https://github.com/scrapinghub/autoextract-poet/pull/13/commits/caf6ad0f3f6855f0eeffe0acc51626014ce62d6d